tools/install_prerequisites.sh

```diff
@@ -100,6 +100,15 @@ fi
 ./tools/ci_download_cockroachdb
 ./tools/ci_download_clickhouse
 
+# Install OPTE
+#
+# OPTE is a Rust package that is consumed by a kernel module called xde. This
+# installs the `xde` driver and some kernel bits required to work with that
+# driver.
+if [[ "${HOST_OS}" == "SunOS" ]]; then
+    ./tools/install_opte.sh
```

Comment:
Should we `pfexec` this?

Reply:
I'd personally rather require users to run the script with `pfexec` itself, but I don't feel strongly.

Reply:
but also, that was a weak preference. If we want to mandate that the entire `install_prerequisites.sh` script be executed with sudo/pfexec, we can do that too

Reply:
I see a `pfexec` in `install_prerequisites.sh:69`, installing the packages?

Reply:
I also don't care too much, I had expected it'd be better for people to have to type `pfexec` to know that the script is...mucking about...but it should be pretty obvious 🤷

Reply:
Done in c217a4fb